{{cookiecutter.github_repository}}/pyproject.toml

```diff
@@ -0,0 +1,147 @@
+version = "{{cookiecutter.version}}"
+description = "{{cookiecutter.project_description}}"
+authors = ["{{cookiecutter.default_from_email}}"]
+
+[tool.poetry.dependencies]
+python = "^3.9"
+Django = "^3.2.10"
+six = "^1.16.0"
+django-environ = "^0.8.1"
+django-sites = "^0.11"
+argon2-cffi = "^21.3.0"
+python-dotenv = "^0.19.2"
+django-cors-headers = "^3.10.1"
+{% if cookiecutter.enable_whitenoise.lower() == 'y' -%}
+whitenoise = "^5.3.0"
+{%- endif %}
+# Extensions
+# -------------------------------------
+pytz = "^2021.3"
+# Models
+# -------------------------------------
+psycopg2-binary = "^2.9.2"
+Pillow = "^8.4.0"
+django-extensions = "^3.1.5"
+django-uuid-upload-path = "^1.0.0"
+django-versatileimagefield = "^2.2"
+# REST APIs
+# -------------------------------------
+djangorestframework = "^3.13.0"
+django-rest-swagger = "^2.2.0"
+# Documentation
+# -------------------------------------
+mkdocs = "^1.2.3"
+# Raven is the Sentry client
+# --------------------------
+{% if cookiecutter.add_sentry == "y" -%}
+raven = "^6.10.0"
+{%- endif %}
+# LOGGING
+# -------------------------------------
+django-log-request-id = "^1.7.0"
+# Async Tasks
+# -------------------------------------
+{%- if cookiecutter.add_celery.lower() == "y" %}
+celery = {extras = ["redis"], version = "^5.2.1"}
+{%- endif %}
+# Auth Stuff
+# -------------------------------------
+PyJWT = "^2.3.0"
+django-mail-templated = "^2.6.5"
+# Production settings
+# =====================================
+# Static Files and Media Storage
+# -------------------------------------
+django-storages = "^1.12.3"
+boto3 = "^1.20.24"
+# Caching
+# -------------------------------------
+django-redis = "^5.1.0"
+hiredis = "^2.0.0"
+{%- if cookiecutter.add_django_auth_wall.lower() == 'y' %}
+# Basic Auth Protection
+# -------------------------------------
+django-auth-wall = "^0.2.0"
+{%- endif %}
+{%- if cookiecutter.add_heroku.lower() == 'y' %}
+uWSGI = "^2.0.16"
+{% endif %}
+{% if cookiecutter.add_newrelic == 'y' %}
+# Logging
+# -------------------------------------
+newrelic = "^7.2.4.171"
+{% endif %}
+
+[tool.poetry.dev-dependencies]
+{%- if cookiecutter.add_pre_commit.lower() == 'y' %}
+pre-commit = "^2.16.0"
+{%- endif %}
+{% if cookiecutter.add_ansible.lower() == 'y' %}
+ansible = "^4.7.0"
+{%- endif %}
+# Documentation
+# -------------------------------------
+isort = "^5.10.1"
+black = "^21.12b0"
+flake8 = "^4.0.1"
+# Debugging
+# -------------------------------------
+django-debug-toolbar = "^3.2.3"
+ipdb = "^0.13.9"
+# Testing and coverage
+# -------------------------------------
+pytest = "^6.2.5"
+pytest-django = "^4.5.2"
+pytest-cov = "^3.0.0"
+django-dynamic-fixture = "^3.1.2"
+pytest-mock = "^3.6.1"
+mypy = "^0.910"
+django-stubs = "^1.9.0"
+# Versioning
+# -------------------------------------
+bump2version = "^1.0.1"
+
+[build-system]
+requires = ["poetry-core>=1.0.0"]
+build-backend = "poetry.core.masonry.api"
+
+[tool.isort]
+line_length = 120
```

Comment:
We should tell `black` as well that line_length should be 120. Otherwise isort and black will fight for imports exceeding >90 chars.

Or we can use isort provided *black config* https://pycqa.github.io/isort/docs/configuration/black_compatibility.html

Reply:
Agreed, that said, let's try to go with minimal changes from the default values, and for the shake of standardization let's stick with 89 char limits as earlier across the broard, and use `profile = black` here and remove the duplicated isort configs from here. 

isort Black profile provides these settings by default: 
https://github.com/PyCQA/isort/blob/main/isort/profiles.py#L4-L10

Reply:
@theskumar Fixed this and removed redundant config as well as changes.